fv3core/examples/standalone/runfile/acoustics.py

```diff
@@ -39,20 +39,6 @@ def initialize_serializer(data_directory: str, rank: int = 0) -> serialbox.Seria
     )
 
 
-# Only used if reading the grid from savepoints on disk
-# Default is to generate a baroclinic initialization
-def read_grid(serializer: serialbox.Serializer, rank: int = 0) -> Grid:
-    """Uses the serializer to generate a Grid object from serialized data"""
-    grid_savepoint = serializer.get_savepoint("Grid-Info")[0]
-    grid_data = {}
-    grid_fields = serializer.fields_at_savepoint(grid_savepoint)
-    for field in grid_fields:
-        grid_data[field] = serializer.read(field, grid_savepoint)
-        if len(grid_data[field].flatten()) == 1:
-            grid_data[field] = grid_data[field][0]
-    return fv3core.testing.TranslateGrid(grid_data, rank).python_grid()
-
```

Comment:
actually we'd prefer to not delete this yet -- it still might be used for running non-baroclinic configurations.

Reply:
Would you say after these tests running a few days that it's stable enough to remove this code from the file, and use the git history to recover it if we need it? If not I can put it back, in which case when should we re-assess this?

Reply:
let's remove it yeah.